utils/linkage.py

```diff
@@ -6,6 +6,50 @@
 """
 
 
+def get_address_line_1_from_full_address(address: str) -> str:
+    """Given a full address, return the first line of the formatted address
+
+    Address line 1 usually includes street address or PO Box information.
+
+    Args:
+        address: raw string representing full address
+    Returns:
+        address_line_1
+
+    Sample Usage:
+    >>> get_address_line_1_from_full_address("6727 W. Corrine Dr.  Peoria,AZ 85381")
+    "6727 W. Corrine Dr."
```

Comment:
failing tests because this is a string and not object -> just adjust to single quotes